specification/hybridkubernetes/resource-manager/Microsoft.Kubernetes/GA/2020-02-01/connectedClusters.json

```diff
@@ -0,0 +1,740 @@
+            },
+            {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+            },
+            {
+              "$ref": "#/parameters/ClusterNamePathParam"
+            }
+          ],
+          "responses": {
+            "200": {
+              "description": "Resource retrieved successfully.",
+              "schema": {
+                "$ref": "#/definitions/ConnectedCluster"
+              }
+            },
+            "default": {
+              "description": "Error response describing why the operation failed.",
+              "schema": {
+                "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+              }
+            }
+          }
+        },
+        "delete": {
+          "summary": "Delete a connected cluster.",
+          "operationId": "ConnectedCluster_Delete",
+          "description": "Delete a connected cluster, removing the tracked resource in Azure Resource Manager (ARM).",
+          "produces": [
+            "application/json"
+          ],
+          "x-ms-examples": {
+            "DeleteClusterExample": {
+              "$ref": "./examples/DeleteClusterExample.json"
+            }
+          },
+          "parameters": [
+            {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+            },
+            {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+            },
+            {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+            },
+            {
+              "$ref": "#/parameters/ClusterNamePathParam"
+            }
+          ],
+          "responses": {
+            "200": {
+              "description": "OK"
+            },
+            "202": {
+              "description": "Resource deletion is in progress"
+            },
+            "204": {
+              "description": "The resource was not found but the request is well formed"
+            },
+            "default": {
+              "description": "Resource Provider error response describing why the operation failed.",
+              "schema": {
+                "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+              }
+            }
+          },
+          "x-ms-long-running-operation": true,
+          "x-ms-long-running-operation-options": {
+            "final-state-via": "location"
+          }
+        }
+      },
+      "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.Kubernetes/connectedClusters/{clusterName}/listClusterUserCredentials": {
```

Comment:
listClusterUserCredentials should not be defined in GA api version.

Reply:
my bad, fixing it